CRM/Contact/Form/Task/PDFLetterCommon.php

```diff
@@ -37,6 +37,20 @@
 class CRM_Contact_Form_Task_PDFLetterCommon {
 
   /**
+   * @return array
+   *   Array(string $machineName => string $label).
+   */
+  public static function getLoggingOptions() {
+    return array(
+      'none' => ts('Do not record'),
+      'multiple' => ts('Multiple activities (one per contact)'),
+      'combined' => ts('One combined activity'),
+      'combined-attached' => ts('One combined activity plus one file attachment'),
+      // 'multiple-attached' <== not worth the work
```

Comment:
Valid point. I guess it's a cautionary note for someone who goes in thinking that they'll implement "multiple activities plus file attachments". (I briefly considered two orthogonal settings for `multiple`/`combined` and `attached`/`not-attached`, but moved to one setting because the combinations weren't equally sensible.)